circuits/circom/prover/wasm/witness_calculator.js

```diff
@@ -0,0 +1,330 @@
+module.exports = async function builder(code, options) {
```

Comment:
wasn't it possible to ts code here?
Or you didn't want to bother about the typings?

Reply:
autogenerated, as above https://github.com/privacy-scaling-explorations/e2e-zk-ecdsa/pull/83#discussion_r1133407276

I would like typescript here, but it comes out of the code generator as plain js.